src/main.scss

```
@@ -0,0 +1,113 @@
+body {
+  margin: 8px;
+  @font-face {
+    font-family: "Avenir";
+    src: url(./fonts/Avenir-Book.ttf);
+    font-weight: 300;
+    font-style: normal;
+  }
+  @font-face {
+    font-family: "Avenir";
+    src: url(./fonts/Avenir-Heavy.ttf);
+    font-weight: 700;
+    font-style: normal;
+  }
+}
+
+.search-form {
+  display: flex;
+  justify-content: center;
+  margin: 20px auto;
+
+  &--long {
+    width: 100%;
+    height: 70px;
+  }
+  &--short {
+    width: 100%;
+    height: 42px;
+  }
```

```suggestion
.search-form {
  display: flex;
  justify-content: center;
  margin: 20px auto;
  width: 100%;
  height: 70px;
  
  &--short {
    height: 42px;
  }
```
Example how will change styles after fixes html.
we need to change only height for small form.
please fix similar cases for all elements.